src/AccessionTables/test/Table.test.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { screen, within } from '@testing-library/react';
+import user from '@testing-library/user-event';
+import { byRole } from 'testing-library-selector';
```

Comment:
Did not know about this. Thanks for adding to the review. I am not kinda into the topic but some of your recipes I found useful.